README.md

```diff
@@ -871,6 +871,7 @@ This includes libraries for things like XML, JSON, CSV, and other similar format
 ### Others ###
 
 * [libbson][235] - BSON utility library. [``Apache-2.0``][Apache-2.0]
+* [libcbor][573]: CBOR protocol implementation for C and others  
```

Comment:
Can you add a link to the MIT license here, as per the other entries, @kelvinhammond ?

Reply:
updated, thanks.